_posts/2023-01-12-install-jekyll-and-bundler.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Guide for installing Jekyll & Bundler
+description: Learn how to install Jekyll & Bundler in an Ubuntu container 
+date: 2023-01-12
+layout: article
+authors: 
+    - gonzoknows
+published: true
+---
+
```

Comment:
I will review this later today. Nice work.

Reply:
> I will review this later today. Nice work.
> 
> I will add in some alternative methods too.
> 
> Also, If possible create a new PR for posts in future in that way the existing one can be merged easily.

how do you do that?

Reply:
> > I will review this later today. Nice work.
> > 
> > I will add in some alternative methods too.
> > 
> > Also, If possible create a new PR for posts in future in that way the existing one can be merged easily.
> 
> How do you do that? I was trying to do that here, lol
> ![Screenshot from 2023-01-12 17-44-49](https://user-images.githubusercontent.com/122128828/212217980-fcb16f36-5c07-461f-be96-54d38e176a7c.png)
> 

Create a new branch with handbook main branch as base repo , push changes to it using git. Or upload files directly using the GitHub interface.

Note: If you are trying to clone it locally you need to use `git clone <repo-url> -b <branch>` instead of normal `git clone`.

Reply:
Oh, you have to create a separate branch. Thanks